### PR TITLE
Axios

### PR DESCRIPTION
**Pull request by Guy-deMessimy**

### CODE REVIEW

**Review by gaetan-cordonnier — DISMISSED** [review]
Good

File: src/pages/admin/DatesAdmin/EditDateAdmin.jsx [hunk 1]
```diff
@@ -5,72 +5,97 @@ import { ContainerEditDate } from "./style.jsx";
...
       const updDate = {
         id: current.id,
         date,
-        message,
+        datefin,
+        text,
       };
 
       updateDate(updDate);
+      console.log(updDate);
```
Comment (LoicBrassart) [inline src/pages/admin/DatesAdmin/EditDateAdmin.jsx]: Here

File: src/pages/admin/DatesAdmin/EditDateAdmin.jsx [hunk 2]
```diff
@@ -5,72 +5,97 @@ import { ContainerEditDate } from "./style.jsx";
...
-      const newDate = {
+      const newAtelier = {
         date,
-        message,
+        datefin,
+        text,
       };
-      AddDate(newDate);
+      AddDate(newAtelier);
+      console.log(newAtelier);
```
Comment (LoicBrassart) [inline src/pages/admin/DatesAdmin/EditDateAdmin.jsx]: Here

File: .env.sample [hunk 3]
```diff
@@ -0,0 +1,6 @@
+//Configuration https://www.emailjs.com/
```
Comment (LoicBrassart) [inline .env.sample]: No comments in .env

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Remove console.log plz

**Review by gaetan-cordonnier — APPROVED** [review]
Console.log deleted

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**